.github/workflows/ci.yml

```diff
@@ -50,42 +44,15 @@ jobs:
       run: rustup show
     - uses: olix0r/cargo-action-fmt@ee1ef42932e44794821dab57ef1bf7a73df8b21f
     - uses: actions/checkout@v2
-    - name: run cargo check
+    - uses: extractions/setup-just@v1
+    - name: run just check
       run: |
-        cargo check \
-          --workspace \
-          --all-features \
-          --quiet \
-          --message-format=json |
-          cargo-action-fmt
+        just check
```

Comment:
nit: this could just be
```yaml
- run: just check
```
now, the `|` is only needed when the string is multiline, and if we exclude the `name`, the name of the step is just the `run:` command.